tests/test_module.py

```diff
@@ -115,3 +119,172 @@ def test_responses(eval_paths, config) -> None:
     assert isinstance(response_dict, dict)
     assert len(response_dict) == config.n_objects
     assert len(response_dict["object_1"]) == config.n_attributes
+
+
+def test_heatmaps_for_current_model(plot_paths_fixture, config) -> None:
+    """Test that the model in the config has folders and files saved in the plots folder."""
+    # Get the plotting path
+    plots_path = plot_paths_fixture[0]
+
+    # Get the path to the model currently in the config
+    model = config.model
+    model_folder = plots_path / model
+
+    # Check that the model folder exists
+    assert model_folder.exists()
+    assert model_folder.is_dir()
+
+    # Check that the model folder contains a subfolder for the number of evaluated red herring clues
+    n_red_herring_clues_evaluated = config.n_red_herring_clues_evaluated
+    red_herring_folder = model_folder / f"{n_red_herring_clues_evaluated}rh"
+    assert red_herring_folder.exists()
+    assert red_herring_folder.is_dir()
+
+    # Check that the folder contains the expected cell score file
+    n_puzzles = config.n_puzzles
+    cell_score_plot_filename = f"mean_cell_score_{model}_{n_red_herring_clues_evaluated}rh_{n_puzzles}_puzzles.png"
+    cell_score_plot_file_path = red_herring_folder / cell_score_plot_filename
+
+    assert cell_score_plot_file_path.exists()
+    assert cell_score_plot_file_path.is_file()
+    assert os.path.getsize(cell_score_plot_file_path) > 0
```

Comment:
I would split this up in (at least) 3 distinct tests. You could collect these tests together in a `TestHeatmapsForCurrentModel` class, which `pytest` supports - helps to keep the tests organised.